docs/user-guide/expressions/user-defined-functions.md

```diff
@@ -45,9 +41,9 @@ df.with_columns([
 ])
 ```
 
-Use cases for `map` in the `group_by` context are slim. They are only used for performance reasons, but can quite easily lead to incorrect results. Let me explain why.
+Use cases for `map_batches` in the `group_by` context are slim. They are only used for performance reasons, but can quite easily lead to incorrect results. Let me explain why.
```

Comment:
to be honest I don't really understand this phrase to begin with - what are the performance reasons to use `map_batch`? Or is that only on the Rust side, referring to `map`?

Reply:
You could do an elementwise operations with map batches. E.g. `lambda x * 2` would be correct in both.